src/main/java/frc/robot/subsystems/vision/Vision.java

```diff
@@ -36,6 +38,15 @@ public List<VisionIO.ScoreParameters> getScoreParameters() {
         return params;
     }
 
+    public OptionalDouble getYawToNote() {
+        for (VisionModule module : modules) {
+            if (module.getYawToNote().isPresent()) {
+                return module.getYawToNote();
+            }
+        }
+        return OptionalDouble.empty();
```

Comment:
I'd just remember which one is the RKNN module and use this one instead of looping